.github/workflows/test.yml

```diff
@@ -75,97 +76,50 @@ jobs:
         # https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#expanding-or-adding-matrix-configurations
         include:
           # test default settings with 3.11 across all supported evm versions
-          - python-version: ["3.11", "311"]
-            debug: false
-            opt-mode: gas
-            evm-version: london
-            evm-backend: revm
-
-          - python-version: ["3.11", "311"]
-            debug: false
-            opt-mode: gas
-            evm-version: paris
-            evm-backend: revm
-
-          # redundant rule, for clarity
-          - python-version: ["3.11", "311"]
-            debug: false
-            opt-mode: gas
-            evm-version: shanghai
-            evm-backend: revm
-
-          - python-version: ["3.11", "311"]
-            debug: false
-            opt-mode: gas
-            evm-version: cancun
-            evm-backend: revm
+          - evm-version: london
+          - evm-version: paris
+          - evm-version: cancun
 
           # py-evm rules
-          - python-version: ["3.11", "311"]
-            debug: false
-            opt-mode: codesize
-            evm-version: london
+          - evm-version: london
```

Comment:
honestly, not sure why we should test py-evm + london. if we are going to have two py-evm runs, it should be cancun + shanghai

Reply:
```suggestion
          - evm-version: shanghai
```